types.ts

```diff
@@ -16,3 +18,18 @@ export type Meetup = {
     past: string;
   };
 };
+
+export type MeetupEvent = {
+  id: string;
+  title: string;
+  date: string;
+  description: string;
+  location: string;
+  link: string;
+};
+
+export enum meetups {
+  'codegreenlondon' = 'codegreenlondon',
+  'reactnativelondon' = 'reactnativelondon',
+  'default' = 'defaultContent',
```

Comment:
[Code quality] You shouldn't need to have the same value on both side of the equality (can just have it once). Watch out for the capitalisation